azimuth_capi/tests/test_operator.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+from unittest import mock
+
+import kopf
+
+from azimuth_capi import operator
+from azimuth_capi.models import v1alpha1 as api
+
+class TestOperator(unittest.IsolatedAsyncioTestCase):
```

Comment:
Does this need to be async?

Reply:
this one didn't need to be in the end, but the usually do need to be.